tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,139 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+setupI = {
```

Comment:
@yeganer Oh, then I cannot think of any way of setting `reference_path`. And, apart from that , saving reference file for all the generated permutations ( even assuming we have 4 options to set(i.e. `ionization`, `excitation`, etc), and having 2 different values that can be set for each option - 2^4 = 16 reference files(the real no. will be much greater than that, considering all options).